spec/jobs/problem_checks_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Jobs::ProblemChecks do
+  before do
+    AdminDashboardData.reset_problem_checks
```

Comment:
I think this should be called in an `after` block. If we're calling it in a `before` block means we have a leaky state else where or we will be preventing the discovery of a leaky state in the future.

Reply:
I never know when to use `before` or `after` for these kind of things, I guess `after` makes more sense because other specs can run after this one :+1: